resources/python/pipelines/pipelines.yml

```diff
@@ -0,0 +1,179 @@
+# Copyright 2023 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+
+parameters:
+  branch: ""
+
+jobs:
+- job: install_source_dependencies_3_7
+  displayName: Install source dependencies Python 3.7
+  condition: not(endsWith( variables['System.PullRequest.SourceBranch'], '-automated'))
+  pool:
+    vmImage: 'macOS-12'
+  steps:
+  - template: template.yml
+    parameters:
+      python_version: 3.7
+  - script: |
+      git checkout $(System.PullRequest.SourceBranch)
+      git checkout -b ${{ parameters.branch }}
```

Comment:
You can combine these 2 lines into one:
```yml
git checkout -b ${{ parameters.branch }} $(System.PullRequest.SourceBranch)
```

You might also want to use `git switch` instead if these VMs are using a recent enough version of GIT.

Not important or mandatory at all ;)

Reply:
Good call!